src/app/public/modules/autocomplete/autocomplete.component.html

```diff
@@ -3,7 +3,7 @@
   aria-haspopup="listbox"
   class="sky-autocomplete"
   role="combobox"
-  [attr.aria-expanded]="isOpen"
+  [attr.aria-expanded]="isOpen && (searchText || showActionsArea)"
```

Comment:
The trick here is that we have a state where the dropdown is technically "open" but nothing is shown because we aren't searching anything at the time. This has always been a case but our a11y tests just weren't appropriately catching it due to that state not being hit. With my changes - the a11y test caught it and showed this issue. This extra `and` case catches this case and marks the `combobox` appropriately.

Reply:
Going to reevaluate a bit - I think my changes might be making `isOpen` less reliable.